client/components/AddReview/DesktopNewPost.jsx

```diff
@@ -195,11 +200,11 @@ const DesktopAddNewPost = () => {
                                         </motion.div>
                                     ))}
                                 </Carousel>
-                                <Button
-                                    onClick={() => handelAnimation()}
-                                    sx={{ textTransform: "none", backgroundColor: "#088395", color: "white", m: 2, p: 1 }}>
-                                    Next <ArrowForwardIosIcon /><ArrowForwardIosIcon /><ArrowForwardIosIcon />
-                                </Button>
+                                <button
+                                    className="critique-button"
+                                    onClick={() => handelAnimation()}>
```

Comment:
not sure if we care, but handle is misspelt

Reply:
I think we should change it. I'll merge it over for now, but we can fix it in the next update.